LICENSE

```diff
@@ -1,23 +0,0 @@
-MIT License
-
-Copyright (c) 2014 Richard Barlow, Jeremy Morse
-              2015 Jeremy Morse
```

Comment:
We [need](https://opsmanual.studentrobotics.org/about-the-charity/vision-mission-and-values#open-by-default) an open source licence for this firmware please.

I think there's also a pretty good argument that this is a derivative work as you were looking at the old source code when writing this, so I'd suggest keeping the attributions in the current licence.